ipmiMgrApp/Db/fru_common_alias.template

```diff
@@ -23,3 +23,22 @@ alias("$(dev):$(id)$(unit):PSN", "$(P)$(R)$(id)ProdSN-Cte")
 alias("$(dev):$(id)$(unit):PPARTNUMBER", "$(P)$(R)$(id)ProdPN-Cte")
 
 alias("$(dev):$(id)$(unit):POWERCTL", "$(P)$(R)$(id)PowerCtl-Sel")
+
+record(bo, "$(P)$(R)$(id)SoftReset-Cmd") {
+  field(DESC, "Power cycle system")
+  field(FLNK, "$(P)$(R)$(id)SoftReset1")
+  field(PINI, "NO")
+}
+record(longout, "$(P)$(R)$(id)SoftReset1") {
+  field(DESC, "Power off system")
+  field(DOL, "0")
+  field(OUT, "$(P)$(R)$(id)PowerCtl-Sel PP")
+  field(FLNK, "$(P)$(R)$(id)SoftReset2")
+}
+record(calcout, "$(P)$(R)$(id)SoftReset2") {
+  field(DESC, "Power on system")
+  field(CALC, "1")
+  field(OOPT, "Every Time")
+  field(ODLY, "15")
+  field(OUT, "$(P)$(R)$(id)PowerCtl-Sel PP")
+}
```

Comment:
I wonder if it is better to use [`seq` record][1] instead. It would be a single record this way. However, it would not allow you to be sure the first processing has ended before writing the second value. [`sseq` record][2] allows this through the WAIT fields, but it would make the IOC depend on calc.

[1]: https://epics.anl.gov/base/R7-0/7-docs/seqRecord.html
[2]: https://epics-modules.github.io/calc/sseqRecord.html

Reply:
@gustavosr8, from our in-person discussion, you'll describe that why you won't be using `seq` nor `sseq` in the commit message instead?